.github/workflows/helm-release.yaml

```diff
@@ -0,0 +1,59 @@
+name: Release Charts
+
+on:
+  push:
+    paths:
+      - "charts/perses/**"
+    branches:
+      - main
+
+jobs:
+  release:
+    permissions:
+      contents: write # to push chart release and create a release (helm/chart-releaser-action)
+      packages: write # needed for ghcr access
+      id-token: write # needed for keyless signing
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3.5
+        with:
+          version: v3.12.0
```

Comment:
perhaps we could leave it empty to always get the latest Helm version

Reply:
btw to be coherent we should align the version with the linter workflow: https://github.com/perses/perses/blob/main/.github/actions/setup_helmlint/action.yaml#L14

So either upgrade the version, or remove it so we can get the latest one everytime